### DISCUSSION TITLE
get_trends_place() functionality

### DISCUSSION THREAD

**Discussion by jgbuck** [tweepy/tweepy#2116] [Questions] [answered]
Hi, I'm trying to collect trends using some code that used to work before the paid update to the Twitter API. I have v2 API access yet I can't get the following code to work:

auth = tweepy.OAuthHandler(consumer_key, consumer_secret)
auth.set_access_token(access_token, access_token_secret)
api = tweepy.API(auth)
trends = api.get_place_trends(<some location number>)

It gives the following error: 
Forbidden: 403 Forbidden
453 - You currently have access to Twitter API v2 endpoints and limited v1.1 endpoints only. If you need access to this endpoint, you may need a different access level.

Yet when I look at the API v2 documentation on the Twitter Dev platform it says a trends feature is "COMING SOON", so is the get_trends_place() functionality obsolete currently, does the Basic v2 not cover this functionality, or am I doing something wrong?

Thanks in advance

**Comment by Harmon758** [comment, marked as answer]
`tweepy.API` is Tweepy's interface for Twitter API v1.1.
[Access to Twitter API v1.1 has been deprecated](https://twittercommunity.com/t/announcing-new-access-tiers-for-the-twitter-api/188728).
You're correct in that Twitter API v2 does not currently have any endpoints relating to trends.

For code block usage, see https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks.
